mugoh/qstioner-v2
# #163217123 Import Files from V1 to Setup Project Structure

#### What does this PR do?
Create project structure for the project Version two

#### Description of Task to be completed?
- Import required files from Version 1 of the project
- Have the project structure setup

#### How should this be manually tested?
You can view the model by:
1. Clone this repo into your machine
``` shell
$ git clone https://github.com/hogum/qstioner-api.git
```
2. Checkout this chore branch:
``` shell
$ git checkout ch-project-structure-163217123
```
3. Navigate to the repo directory
``` shell
$ cd qstioner
```
4. You should be able to see the project directories  and pre-existing files imported from v1

Testing will be made possible with a  user view completed.
- Access the repo on github [here](https://github.com/hogum/qstioner-v2)
- Switch to the branch ch-project-structure-163217123
- The project structure will be visible

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[163217123](https://www.pivotaltracker.com/story/show/163217123)
